utils/runtime_test.go

```diff
@@ -14,51 +14,46 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var pgURLString string
-var rdURLString string
-var foundPgString bool
-var foundRdString bool
-
-func initTest() {
-	//get db and redis connection strings
-	pgURLString, foundPgString = os.LookupEnv("TEST_DB_URL")
-	rdURLString, foundRdString = os.LookupEnv("TEST_RD_URL")
-	if foundPgString == false || foundRdString == false {
-		panic(fmt.Sprintf("TEST_DB_URL and TEST_RD_URL env vars need to be set before running test"))
-	}
-}
-
 func TestCheckRuntimeDB(t *testing.T) {
-	initTest()
+	config.InitDB()
 	//setup connection (postgres) object
+	pgConMaps :=  map[string]string{"DB_URL_STATUS":config.DbStatusURL, "DB_URL_GUARD": config.DbGuardURL}
 	var connection = db.ConnectionUrls{
-		PgURL: pgURLString,
+		PgURL: pgConMaps,
 		RdURL: "",
 	}
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
 	shared := new(sharedpb.SignedRuntimeInfoRequest)
 	runtimeInfoReportPass, _ := CheckRuntime(ctx, shared, connection)
-	assert.Equal(t, runtimeInfoReportPass.DbStatusExtra, []byte(constant.DBConnectionHealthy), "DB is not running")
+	assert.True(t, strings.Contains(string(runtimeInfoReportPass.DbStatusExtra[0]), constant.DBConnectionHealthy), "connection not successful")
+	assert.True(t, strings.Contains(string(runtimeInfoReportPass.DbStatusExtra[1]), constant.DBConnectionHealthy), "connection not successful")
+
 	//disable connection string
-	connection.PgURL = ""
-	runtimeInfoReportFail, _ := CheckRuntime(ctx, shared, connection)
-	assert.Equal(t, runtimeInfoReportFail.DbStatusExtra, []byte(nil), "DB connection is still provided, error!")
+	pgEmptyConMaps :=  map[string]string{"DB_URL_STATUS":"", "DB_URL_GUARD": ""}
+	var emptyConnection = db.ConnectionUrls{
+		PgURL: pgEmptyConMaps,
+		RdURL: "",
+	}
+	runtimeInfoReportFail, _ := CheckRuntime(ctx, shared, emptyConnection)
```

Comment:
why not check for err (almost all errors are ignored in this file, which is exactly what tests are for, right?)

Reply:
correct.